source/opt/local_access_chain_convert_pass.cpp

```diff
@@ -236,6 +241,16 @@ void LocalAccessChainConvertPass::FindTargetVars(Function* func) {
   }
 }
 
+bool LocalAccessChainConvertPass::HasNoOpenCL100DebugRef(
```

Comment:
Naming a function with a negative in it is more awkward to read because you can end up with double negaives:

`if (!HasNoDebugRef)` reads "if it does not have no debug ref."

Reply:
Ah .. the name is wrong. I changed it to `HasOpenCL100DebugRef`.